projects/1.20/assets/mowzies-mobs/mowziesmobs/lang/zh_cn.json

```
@@ -0,0 +1,402 @@
+  "item.mowziesmobs.foliaath_seed": "巨噬叶芽",
+  "item.mowziesmobs.mob_remover": "生物移除器",
+  "item.mowziesmobs.wrought_axe": "千锻巨斧",
+  "item.mowziesmobs.wrought_helmet": "精炼头盔",
+  "item.mowziesmobs.umvuthana_mask_fury": "复仇面具",
+  "item.mowziesmobs.umvuthana_mask_fear": "恐惧面具",
+  "item.mowziesmobs.umvuthana_mask_rage": "狂暴面具",
+  "item.mowziesmobs.umvuthana_mask_bliss": "极乐面具",
+  "item.mowziesmobs.umvuthana_mask_misery": "痛苦面具",
+  "item.mowziesmobs.umvuthana_mask_faith": "信仰面具",
+  "item.mowziesmobs.sol_visage": "太阳之面",
+  "item.mowziesmobs.dart": "吹镖",
+  "item.mowziesmobs.spear": "长矛",
+  "item.mowziesmobs.blowgun": "吹筒",
+  "item.mowziesmobs.grant_suns_blessing": "接受太阳祝福",
+  "item.mowziesmobs.test_structure": "结构测试器",
+  "item.mowziesmobs.ice_crystal": "冰霜结晶",
+  "item.mowziesmobs.earth_talisman": "大地护符",
+  "item.mowziesmobs.earthbore_gauntlet": "裂地护手",
+
+  "item.mowziesmobs.blowgun.text.0": "快速装填",
+  "item.mowziesmobs.blowgun.text.1": "短射程",
+  "item.mowziesmobs.blowgun.text.2": "仅能发射吹镖",
+
+  "item.mowziesmobs.wrought_axe.text.0": "坚不可摧",
+  "item.mowziesmobs.wrought_axe.text.1": "右击横扫攻击",
+  "item.mowziesmobs.wrought_axe.text.2": "潜行右击震撼大地并释放冲击波",
+
+  "item.mowziesmobs.sol_visage.text.0": "坚不可摧",
+  "item.mowziesmobs.sol_visage.text.1": "装备后手持其他面具右键以召唤随从",
+  "item.mowziesmobs.sol_visage.text.2": "右键随从可将其变回面具",
+
+  "item.mowziesmobs.umvuthana_mask_fear.text.0": "穿戴后获得 迅捷",
+  "item.mowziesmobs.umvuthana_mask_fear.text.1": "或许还有什么用......?",
+  "item.mowziesmobs.umvuthana_mask_rage.text.0": "穿戴后获得 急迫",
+  "item.mowziesmobs.umvuthana_mask_rage.text.1": "或许还有什么用......?",
+  "item.mowziesmobs.umvuthana_mask_misery.text.0": "穿戴后获得 抗性提升",
+  "item.mowziesmobs.umvuthana_mask_misery.text.1": "或许还有什么用......?",
+  "item.mowziesmobs.umvuthana_mask_bliss.text.0": "穿戴后获得 跳跃提升",
+  "item.mowziesmobs.umvuthana_mask_bliss.text.1": "或许还有什么用......?",
+  "item.mowziesmobs.umvuthana_mask_fury.text.0": "穿戴后获得 力量",
+  "item.mowziesmobs.umvuthana_mask_fury.text.1": "或许还有什么用......?",
+  "item.mowziesmobs.umvuthana_mask_faith.text.0": "穿戴后获得 生命提升",
+  "item.mowziesmobs.umvuthana_mask_faith.text.1": "或许还有什么用......?",
+
+  "item.mowziesmobs.dart.text.0": "仅用于吹筒",
+  "item.mowziesmobs.dart.text.1": "使目标中毒",
+
+  "item.mowziesmobs.foliaath_seed.text.0": "种植在草方块上",
+  "item.mowziesmobs.foliaath_seed.text.1": "每隔半天，巨噬叶幼苗就会变得饥饿",
+  "item.mowziesmobs.foliaath_seed.text.2": "向它丢掷肉来喂养它",
+  "item.mowziesmobs.foliaath_seed.text.3": "经过两天，巨噬叶就会完全长大",
+
+  "item.mowziesmobs.grant_suns_blessing.text.0": "授予太阳祝福",
+  "item.mowziesmobs.grant_suns_blessing.text.1": "分钟",
+  "item.mowziesmobs.grant_suns_blessing.text.2": "在使用能力之后开始计时",
+  "item.mowziesmobs.grant_suns_blessing.text.3": "空手右键以召唤太阳打击",
+  "item.mowziesmobs.grant_suns_blessing.text.4": "空手潜行左键以释放太阳冲击",
+  "item.mowziesmobs.grant_suns_blessing.text.5": "空手潜行右键以发射太阳光束 (消耗",
+  "item.mowziesmobs.grant_suns_blessing.text.6": "分钟)",
+  "item.mowziesmobs.grant_suns_blessing.text.7": "空手潜行并同时点击左键和右键以召唤太阳超新星 (消耗",

```

| **审阅级别** |
|:--------------:|
|❓ **疑问** / 不阻止合并|

原文：
- **With empty hand, sneak, then left and right click at the same time to call a Supernova (costs**
```suggestion
  "item.mowziesmobs.grant_suns_blessing.text.7": "1",
```
supernova只是“超新星”，为什么有太阳